src/tools/web/webd/src/index.js

```diff
@@ -32,12 +32,12 @@ getVersions()
       process.exit(1);
     } else {
       const { major, minor, micro } = versions.elektra;
-      const versionSupported = major >= 0 && minor >= 8 && micro >= 23;
+      const versionSupported = major >= 0 && minor >= 9 && micro >= 0;
```

Comment:
This will stop working for 1.0.0?

Reply:
Yes

Reply:
Please fix as we might forget to update it otherwise.